ddtrace/contrib/flask/middleware.py

```diff
@@ -125,6 +124,7 @@ def _process_response(self, response):
             return
 
         code = response.status_code if response else ""
+        trace_utils.set_http_meta(span, config.flask, status_code=code, response_headers=response.headers)
```

Comment:
This looks to be new functionality which we don't have a test-case for. I'd be ok with pulling it out of this PR as well like you've done with the others 😄